patterns/2-structured/project-setup/communication-tooling.md

```diff
@@ -32,23 +30,23 @@ answer.
 ## Solution
 
 The host team needs to be clear on the benefit of providing company-public,
-archived, searchable, linkable communication channels that are free to subscribe
-to by anyone in the company.
+archived, searchable, linkable communication channels that anyone in the company can subscribe to.
```

Comment:
@MaineC What is the consequence of this?

What will the host team do that shows that they are "clear about the benefits ..."?

Reply:
Maybe it would be clearer to state:
The host team should provide company-public, archived, searchable, linkable communication channels that anyone in the company can subscribe to, as there are measurable benefits to supporting open, written communications channels.

Reply:
Working in @NewMexicoKid's suggestion.